hashing/Cargo.toml

```diff
@@ -19,6 +19,7 @@ hex-buffer-serde = "0.3.0"
 itertools = "0.10.1"
 schemars = "0.8.3"
 serde = "1.0.130"
+thiserror = "1.0.29"
```

Comment:
What do we need this new dependency for?

Reply:
I see now, it allows for adding the `#[error(msg)]` annotation to enum variants.